rundeckapp/grails-app/migrations/core/BaseReport.groovy

```diff
@@ -83,4 +83,41 @@ databaseChangeLog = {
             column(name: "filter_applied", type: '${text.type}')
         }
     }
+    changeSet(author: "rundeckdev", id: "Add-job-uuid-to-base-report") {
+        preConditions(onFail: "MARK_RAN") {
+            not {
+                columnExists(tableName: "base_report", columnName: 'job_uuid')
+            }
+        }
+        addColumn(tableName: "base_report") {
+            column(name: 'job_uuid', type: '${varchar255.type}')
+        }
+    }
+        changeSet(author: "rundeckdev", id: "Populate-base-report-job-uuid", dbms: "!mssql") {
```

Comment:
I think you can use the update code you wrote in the mysql section for all databases. You don't really need two different sql update statements.

Reply:
I left just one